src/components/Profile/Profile.js

```diff
@@ -0,0 +1,139 @@
+import './Profile.css';
+import { useState, useEffect } from 'react';
+// eslint-disable-next-line object-curly-newline
```

Comment:
remove this comment. you may need to change eslint config

Reply:
Done!